src/Config/Configurations/Lambda.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare(strict_types=1);
+
+namespace Momento\Config\Configurations;
+
+use Momento\Config\Configuration;
+use Momento\Config\ReadConcern;
+use Momento\Config\Transport\StaticGrpcConfiguration;
+use Momento\Config\Transport\StaticStorageTransportStrategy;
+use Momento\Logging\ILoggerFactory;
+use Momento\Logging\NullLoggerFactory;
+
+/**
+ * Lambda config provides recommended configuration settings for a lambda environment.
+ */
+class Lambda extends Configuration
+{
+    /**
+     * Provides the latest recommended configuration for a Lambda environment.
+     * This configuration may change in future releases to take advantage of
+     * improvements we identify for default configurations.
+     * @param ILoggerFactory|null $loggerFactory
+     * @return Lambda
+     */
+    public static function latest(?ILoggerFactory $loggerFactory = null): Lambda
+    {
+        return self::v1($loggerFactory);
+    }
+
+    /**
+     * Provides the version 1 configuration for a Lambda environment. This configuration is guaranteed not to change
+     * in future releases of the SDK.
+     * @param ILoggerFactory|null $loggerFactory
+     * @return Lambda
+     */
+    public static function v1(?ILoggerFactory $loggerFactory = null): Lambda
+    {
+        $loggerFactory = $loggerFactory ?? new NullLoggerFactory();
+        $grpcConfig = new StaticGrpcConfiguration(1100);
+        $transportStrategy = new StaticStorageTransportStrategy($grpcConfig, $loggerFactory, self::$maxIdleMillis);
```

Comment:
iirc, the main difference in the lambda config was that keepalives were turned off. should we also add the keepalive setting to the grpc config and turn them off here?

Reply:
Yeah, the keepalives were causing errors with infrequently invoked lambdas. If you want to add that in a future pr, we should get rid of the v1 config for now and only use latest.

Reply:
Good to know, thanks. I'll add that in.

Reply:
Ok, I added the keepalive params into the gRPC config before realizing that keepalive is opt-in. The nodejs `Lambda` configuration just omits the keepalive parameters to use the gRPC defaults. So the `Lambda` config is good to go as-is and we're free to add keepalives to new configs and new versions of the existing configs in the future as we see fit.